browser.js

```diff
@@ -107,6 +107,28 @@ define(function(require, exports, module) {
             
             iframe.src = url;
         }
+
+        function setTitle(session, tab, path) {
+            // remember default title between reloads
+            if (!session.doc.title) {
```

Comment:
Hm, I suppose we could do something like

```
tab.title = session.doc.title || (session.doc.title = "[B] " + path);
// don't think we can set new properties on doc?
```

to have the title cached as part of document's state, and therefore set when tab is created (even if hasn't become active yet), but normally title should be updated if the user visits a different URL (and default title is in use)? This wouldn't be the case, if we follow this approach, because the second time this line is run, `session.doc.title` will be set, but to `"[B] old path"`. In fact, it seems I forgot to cache updated title, so that it's set back after reload.

So basically there needs to be a way to know whether default title is in use, in which case we'd update the document's and tab's titles accordingly.

No need for this for our particular use case, though, since we don't change title, and if we need to, we would probably be able to do that manually.

Just pushed a couple of commits to [remove](https://github.com/c9/c9.ide.preview.browser/pull/3/commits/339951a5db1528fe154bada905bfccd33a81bfee) tooltip caching (per your comment) and [remember](https://github.com/c9/c9.ide.preview.browser/pull/3/commits/84eab474877d4adca0b04025b5b960efc8833e77) updated title between reloads. I also renamed the property to `defaultTitleSet` to better fit for a boolean, if not too long? Open to better suggestions!

Thanks again!